client/lib/common/src/data/resource/resource-store.js

```diff
@@ -80,6 +113,12 @@ class ResourceStore extends Store {
         });
     }
 
+    receiveScopeApplications([id, applications]) {
```

Comment:
I know I'm myself not the best example, but please try to add some JSDoc to functions.